roles/memcached/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+memcached:
+  gem:
+    url: https://rubygems.org/downloads/memcached-1.8.0.gem
+    name: memcached-1.8.0.gem
```

Comment:
nah, they only need to be vars so I can override to point to mirror.  don't want to overthink it.